src/hdtPhysicsProfiler.cpp

```
@@ -18,507 +18,518 @@ namespace hdt::physicsprofiler
+			} else {
+				logger::info("{:<80} {:>12.3f} ms {:>10.3f} ms/frame {:>17} {:>8.2f}% {:>8}", a_scope, totalMs, msPerFrame, "-", pct, "-");
+			}
 		}
-	}
 
-	static void logTreeRecursive(const AggregateNode& a_node, std::string a_prefix, std::uint32_t a_depth, std::uint64_t a_frames, const DumpOptions& a_options)
-	{
-		if (a_depth >= a_options.m_maxDepth) {
-			return;
-		}
+		void logTreeRecursive(const AggregateNode& a_node, std::string a_prefix, std::uint32_t a_depth, std::uint64_t a_frames, const DumpOptions& a_options)
```

Cppcheck (2.20.0)

[performance] Function parameter 'a_prefix' should be passed by const reference.

(passedByValue)